templates/travel/partials/hero/hero.snip.html

```diff
@@ -0,0 +1,164 @@
+<!-- Hero -->
+{{#partials.hero}}
```

Comment:
I dont see any changes here wrt to my comments?

Reply:
Fair, let me explain. I changed the markup to: 
```
<section>
  <header>H1  IN HERE</header>
  <div>CONTENT HERE</div>
</section>
```
but left the image outside of the section. I figure this is fine for the sake of screen readers since the images in this hero section are basically background images anyway, and thus not important content for a11y purposes.